modules/virt-installing-fusion-access-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-fusion-access-operator_{context}"]
+= Installing the {IBMFusionSAN} Operator
+
+Install the {IBMFusionSAN} Operator from the *OperatorHub* in the {product-title} web console. Once you have installed the Operator, create the `FusionAccess` custom resource (CR) to complete the installation.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have a working registry enabled.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+
+. In the *Filter by keyword* field, type `IBM fusion for SAN`.
+
+. Select the *{IBMFusionSAN}* tile and click *Install*.
+
+. On the *Install Operator* page, use the default selections for *Update Channel*, *Version*, and *Installation mode*.
+
+. Verify that *Operator recommended Namespace* is selected for *Installed Namespace*. This installs the Operator in the `ibm-fusion-access` namespace.
++
+If this namespace does not yet exist, it is automatically created.
+
+. Verify that *Automatic* is selected for *Update Approval*. *Automatic* is the default value, and enables automatic updates when a new z-stream release is available.
+
+. Click *Install*.
++
+Once the {IBMFusionSAN} Operator installation is successful, a prompt appears to create  a `FusionAccess` custom resource (CR).
+
+. Click *Create FusionAccess*.
+
+. On the *Create FusionAccess* page, enter the object *Name*.
+
+. You can choose to add *Labels* if they are relevant.
+
+. Verify that the *Ibm Cnsa Version* is correct.
+
+. Click *Create*.
+
+The new CR appears in the *FusionAccess* tab of the {IBMFusionSAN} Operator page.
```

Comment:
IMHO I think we should first add the section to create the pull secret. If the user creates the FusionAccess resource without having created the IBM pull secret first, then they will see the FusionAccess in ErrImagePull status because the secret is missing:

![image](https://github.com/user-attachments/assets/7ba081be-335d-429f-a90b-5619e27c643b)

If the user creates the secret afterwards, the FusionAccess does transition to Ready status and all will continue to work, but maybe we don't want them to see an Error status.

Reply:
@mbaldessari can you confirm this can be changed? I seem to remember when we discussed the procedures initially, that we wanted the users to obtain the pull secret after installing the Operator, but before creating the cluster... And the creation of the FusionAccess CR is part of the Operator creation flow, so it wouldn't make sense to break that procedure up.

Can the user obtain the pull secret before installing Operator?

Reply:
Yeah I think I agree with Elsa. Ideally we do: 1. operator install 2. pull secret creation 3. Creation of Fusion Access, that way we do indeed avoid this temporary error state. If we do it this way, there is no real harm, though. Just a bit cleaner maybe